biojava-core/src/test/java/org/biojava/nbio/core/util/CRC64ChecksumTest.java

```diff
@@ -65,16 +65,6 @@ void allbyteRange (){
         assertEquals(allBytesHex, crc64.toString());
     }
 
-    @Test
-    void allRangeIsSameAsAllArray (){
-        byte [] testBytes = new byte [] {1,2,3,4,5};
-        crc64.update(testBytes, 0, testBytes.length);
-        Long valueFromAllRange = crc64.getValue();
-        crc64.reset();
-        crc64.update(testBytes);
-        assertEquals(valueFromAllRange, crc64.getValue());
-    }
-
```

Comment:
Could you explain why this one is removed?

Reply:
because it's a method added in Java9 so won't compile in Java 8. I recently moved from Eclipse to IntelliJ and assumed it was running tests according to Maven setup, sorry I should have just run tests from maven command line before originally committing this.